config/source.go

```diff
@@ -55,23 +57,29 @@ func ParseSource(cfg map[string]string) (Source, error) {
 	sourceConfig := Source{
 		Configuration:  config,
 		OrderingColumn: strings.ToUpper(cfg[OrderingColumn]),
-		KeyColumn:      strings.ToUpper(cfg[KeyColumn]),
 		BatchSize:      defaultBatchSize,
 	}
 
-	if cfg[Columns] != "" {
-		columnsSl := strings.Split(cfg[Columns], ",")
+	if cfg[KeyColumns] != "" {
+		keyColumns := strings.Split(strings.ReplaceAll(cfg[KeyColumns], " ", ""), ",")
+		for i := range keyColumns {
+			if keyColumns[i] == "" {
+				return Source{}, fmt.Errorf("invalid %q", KeyColumns)
```

Comment:
we can just skip it using continue and add sdk.logger warning, what do you think about it?

Reply:
ok, sounds good